release/docker/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM alpine:3.12
+FROM python:3.9-buster
 
-ENV LANG=en_US.UTF-8
+# ENV LANG=en_US.UTF-8
```

Comment:
Why did you remove this?

Reply:
Setting this lang ENV was resulting in some error debug messages. On removing this, it seemed to be working. I believe Alpine Linux needed to set the environment and Debian Buster can work fine without it.

Reply:
If it is not needed, please remove it.